docs/application/web/guides/app-management/web-widget.md

```diff
@@ -1430,90 +1440,116 @@ var parent = document.getElementById('parentDiv');
+Accessing Internet is available through the `XMLHttpRequest()` method. For an example code, see [Communicating with a Web Server](#communicating-with-a-web-server).
 
+<a name="faq25"></a>
 #### Is it possible to implement Web widgets without a Web application?
 
-For your convenience during testing and debugging only, the Web widget engine supports the "Only Web Widget Mode". To distribute Web widgets through the Tizen Store, each Web widget must have at least 1 parent Web application in the package. For more information, see [Application Model](./app-management/web-widget-ww.md#app_model).
+For your convenience during testing and debugging only, the Web widget engine supports the "Only Web Widget Mode". To distribute Web widgets through the Tizen Store, each Web widget must have at least 1 parent Web application in the package. For more information, see [Application Model](#application-model).
 
+<a name="faq26"></a>
 #### How can I determine which APIs are supported in Web widgets?
 
 For the APIs supported by the Web Widget Specification, see [Tizen Wearable Web Widget Specification](https://developer.tizen.org/dev-guide/2.3.2/org.tizen.web.apireference/html/widget_spec/web_widget.html). Additionally, the Tizen Studio provides a Web widget validator that indicates whether a typed-in API follows the Web Widget Specification.
 
+<a name="faq27"></a>
 #### Can Web widgets reuse the JavaScript library or CSS styles of their parent Web application?
 
 Because Web application and Web widget resources are isolated from each other, Web widgets cannot use the JavaScript or CSS files in their parent Web application. The virtual root paths of the Web application and its Web widgets are not the same. You can copy Web application JavaScript and CSS files to the Web widget's folder. However, the Web Widget Specification places more restrictions on the Web widget files than on the Web application files. Therefore, verify all copied Web application JavaScript and CSS files using the Web widget validator.
 
+<a name="faq28"></a>
 #### What are the benefits of Web widgets compared with native widget applications?
 
 Because Web widgets are implemented through the Web Widget Specification, Web-friendly developers can create Web widgets more easily and quickly than native widget applications.
 
+<a name="faq29"></a>
 #### What type of content is best for Web widgets?
 
 Typically as part of a Web application, Web widgets provide their users with important Web application functions (such as launching the parent Web application), and brief, important Web application information. Therefore, simple and lightweight services are best suited for Web widgets.
 
+<a name="faq30"></a>
 #### Can I use CSS `ellipsis`?
 
 Currently, CSS `ellipsis` is not supported. However, the string length can be limiting by using JavaScript. For an example code, see the Scrolling Text sample in [Animation](./app-management/web-widget-ww.md#animation).
```

Comment:
`./app-management/web-widget-ww.md#animation` should be changed into `#animation`

Reply:
done